docs/sources/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1042,6 +1042,38 @@ Status Codes:
           }
         ]
 
+** Example request, with digest information**:
+
+        GET /images/json?digests=1 HTTP/1.1
+
+**Example response, with digest information**:
+
+        HTTP/1.1 200 OK
+        Content-Type: application/json
+
+        [
+          {
+            "Created": 1420064636,
+            "Id": "4986bf8c15363d1c5d15512d5266f8777bfba4974ac56e3270e7760f6f0a8125",
+            "ParentId": "ea13149945cb6b1e746bf28032f02e9b5a793523481a0a18645fc77ad53c4ea2",
+            "RepoDigests": [
+              "localhost:5000/test/busybox@sha256:cbbf2f9a99b47fc460d422812b6a5adff7dfee951d8fa2e4a98caa0382cfbdbf"
+            ],
+            "RepoTags": [
+              "localhost:5000/test/busybox:latest",
+              "busybox:latest",
+            ],
+            "Size": 0,
+            "VirtualSize": 2429728
+          }
+        ]
+
+In the example above, this image ID is associated with 1 digest for the
```

Comment:
@ncdc @jlhawn came by to straighten me out.  I think my rewrite is best for RUX (reader UX) Trying to explain all the components in (RepoTags, RepoDigests, repository, tag, digest)  overloads the sentences to much.  You want to focus the reader on one structure (RepoTags) and then the next (RepoDigest).  Y'all can parse because you have been thinking about it way more than the average reader will.

Just thought of this,  I think part of the confusion is `busybox` appears in both `RepoTags` are they always the same name --- probably huh-- or you can do this:

```
"RepoTags": [
   "localhost:5000/test/busybox:latest",
   "playdate:latest",
 ],
``